raiden/api/rest.py

```diff
@@ -254,7 +256,11 @@ class APIServer(Runnable):  # pragma: no unittest
     _api_prefix = "/api/1"
 
     def __init__(
-        self, rest_api: "RestAPI", config: RestApiConfig, eth_rpc_endpoint: str = None
+        self,
+        rest_api: "RestAPI",
+        config: RestApiConfig,
+        eth_rpc_endpoint: str = None,
+        enable_tracing: bool = False,
```

Comment:
Ideally this would be a flag in `RestApiConfig` (and by extension `RaidenConfig` if / when we add tracing to more parts of Raiden)